Sources/FoundationEssentials/TimeZone/TimeZone_Cache.swift

```diff
@@ -121,14 +121,50 @@ struct TimeZoneCache : Sendable {
         /// Reads from environment variables `TZFILE`, `TZ` and finally the symlink pointed at by the C macro `TZDEFAULT` to figure out what the current (aka "system") time zone is.
         mutating func findCurrentTimeZone() -> TimeZone {
 #if !NO_TZFILE
+#if os(Windows)
```

Comment:
Should we just change this whole section to use `ProcessInfo.processInfo.environment` so that it is portable for both platforms and we don't have to copy this code around?